hokusai/production.yml

```diff
@@ -100,6 +100,91 @@ spec:
                 operator: In
                 values:
                 - foreground
+
```

Comment:
Is there a way to validate these changes before actual deploy? I'm not comfortable doing them blindly.

Reply:
You can run hokusai commands locally to create the staging resources before merging. Happy to hop on a call to do that with you.

Reply:
Also `hokusai staging update --dry-run --skip-checks ...`